package.json

```diff
@@ -520,12 +520,6 @@
         "mac": "cmd+l cmd+m",
         "when": "textInputFocus && editorLangId == 'latex'"
       },
-      {
-        "key": "ctrl+l ctrl+w",
-        "mac": "cmd+l cmd+w",
-        "when": "editorTextFocus && !editorReadonly && editorHasSelection && editorLangId == 'latex'",
-        "command": "latex-workshop.surround"
-      },
```

Comment:
Humm, I do not think `latex-workshop.surround` has ever been deprecated. I think it is only the use of `\` to trigger surrounding that has been deprecated, see #844. I would rather have this feature kept. Tell me if I missed something.

Reply:
You are correct. I mistaken this one with `wrap-env`. Will restore soon.